plugins/tinymcespellchecker.md

```diff
@@ -112,85 +117,99 @@ tinymce.init({
 
 {% include configuration/languages.md %}
 
-### `spellchecker_dialog`
+### `spellchecker_languages`
 
-This option specifies the primary spell checking mode. 
+This option specifies the spellchecker languages that are available to the user, provided as a comma delimited string. The specified languages will be available for the spellchecker when using Spell Checker Pro.
```

Comment:
The end of this paragraph reads a bit awkwardly?

Reply:
It kinda doesn't need the second sentence.

Reply:
removed second sentence.